From 8e680edddd224e16ed3c2c6b605a8695caab39aa Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Sat, 13 Oct 2012 16:18:37 +0300
Subject: [PATCH 1/7] da8xx: Allow use by am33xx based devices

This driver can be used for AM33xx devices, like the popular beaglebone.
---
 drivers/video/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/video/Kconfig b/drivers/video/Kconfig
index b532f02..3a249bb 100644
--- a/drivers/video/Kconfig
+++ b/drivers/video/Kconfig
@@ -2204,7 +2204,7 @@ config FB_SH7760
 
 config FB_DA8XX
 	tristate "DA8xx/OMAP-L1xx Framebuffer support"
-	depends on FB && ARCH_DAVINCI_DA8XX
+	depends on FB && (ARCH_DAVINCI_DA8XX || SOC_AM33XX)
 	select FB_CFB_FILLRECT
 	select FB_CFB_COPYAREA
 	select FB_CFB_IMAGEBLIT
-- 
1.7.7.6

